openedx/core/djangoapps/xblock/utils.py

```diff
@@ -95,23 +95,12 @@ def get_secure_token_for_xblock_handler(user_id, block_key_str, time_idx=0):
 def _get_secure_token_for_xblock_handler(user_id, block_key_str, time_idx: int, hashing_key: str):
     """
     Internal function to extract repeating hashing steps which we
-    call multiple times with different time_idx and hashing key.
+    call mulitple times with different time_idx and hashing peppers.
```

Comment:
```suggestion
    call multiple times with different time_idx and hashing key.
```
I think this change got missed in a merge, and is now behind master's version of the docstring.